api_app/analyzers_manager/migrations/0053_analyzer_config_dns0_rrsets_data.py

```diff
@@ -0,0 +1,232 @@
+from django.db import migrations
+from django.db.models.fields.related_descriptors import (
+    ForwardManyToOneDescriptor,
+    ForwardOneToOneDescriptor,
+    ManyToManyDescriptor,
+)
+
+plugin = {
+    "name": "DNS0_rrsets_data",
+    "python_module": {
+        "module": "dns0.dns0_rrsets.DNS0Rrsets",
+        "base_path": "api_app.analyzers_manager.observable_analyzers",
+    },
+    "description": "Query billions of current and historical DNS resource records sets.",
```

Comment:
sorry another thing, can you add here  a link to the DNS0 service too? That would appear in the GUI as a consequence.